examples/bundle_method_examples/L2TVTV2.jl

```diff
@@ -0,0 +1,68 @@
+using ColorSchemes, Random, QuadraticModels, RipQP, Manifolds, Manopt
```

Comment:
This file should be moved to ManoptExamples.

Reply:
Actually, all of that folder should be removed as it contains outdated examples.